ui/src/containers/NodeInformation.js

```diff
@@ -169,21 +169,24 @@ const NodeInformation = props => {
   );
   const volumeData = volumes.map(volume => {
     const volumePV = pVList.find(
-      pV => pV.metadata.name === volume.metadata.name
+      pV => pV.metadata.name === volume.metadata.name,
     );
-
     return {
       name: volume.metadata.name,
       status:
         (volume && volume.status && volume.status.phase) || STATUS_UNKNOWN,
+      bound:
+        volumePV && volumePV.status && volumePV.status.phase === STATUS_BOUND
```

Comment:
don't know if this is possible, but can it be simplified as: `volumePV?.status?.phase === STATUS_BOUND`?